aws/cloudformation/components/www_redirect.yml.erb

```diff
@@ -37,6 +37,9 @@
     Properties:
       DomainName: www.<%= CDO.pegasus_hostname %>
       ValidationMethod: DNS
+      DomainValidationOptions:
+        - DomainName: www.<%= CDO.pegasus_hostname %>
+          HostedZoneId: !GetAtt [ALB, CanonicalHostedZoneID]
```

Comment:
I think this needs to be the Hosted Zone ID of the base domain name (`code.org` or `cdn-code.org`, etc.)